<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE ProvisioningTransaction PUBLIC "sailpoint.dtd" "sailpoint.dtd">
<ProvisioningTransaction accountDisplayName="CN=stejob1,OU=People,OU=Demo,DC=Eshiam,DC=com" applicationName="Active- Directory" identityDisplayName="Jobs" identityName="stejob1" integration="Active- Directory" name="0000000175" nativeIdentity="CN=stejob1,OU=People,OU=Demo,DC=Eshiam,DC=com" operation="Modify" status="Failed" type="Auto">
  <Attributes>
    <Map>
      <entry key="request">
        <value>
          <AccountRequest application="Active- Directory" nativeIdentity="CN=stejob1,OU=People,OU=Demo,DC=Eshiam,DC=com" op="Modify">
            <Attributes>
              <Map>
                <entry key="provisioningTransactionId" value="c0a8010285b010dc8185cfbe7eff1473"/>
              </Map>
            </Attributes>
            <AttributeRequest name="AC_NewParent" op="Set" value="OU=DisabledUsers,OU=People,OU=Demo,DC=Eshiam,DC=com"/>
            <ProvisioningResult status="failed">
              <Errors>
                <Message key="Error(s) reported back from the IQService - Failed to connect to the server for OU=DisabledUsers,OU=People,OU=Demo,DC=Eshiam,DC=com:There is no such object on the server. 0000208D: NameErr: DSID-03100238, problem 2001 (NO_OBJECT), data 0, best match of: &#x9;'OU=People,OU=Demo,DC=Eshiam,DC=com' . HRESULT:[0x80072030] Possible reasons for failure include a) The Domain Controller is currently not reachable b) The object has either been moved or renamed c) The object has been deleted &#xa; Please Ensure the data has been aggregated before performing the operation " type="Error"/>
              </Errors>
            </ProvisioningResult>
          </AccountRequest>
        </value>
      </entry>
    </Map>
  </Attributes>
</ProvisioningTransaction>